tests/FlatBuffers.Test/FlatBuffersFuzzTests.cs

```diff
@@ -244,17 +264,29 @@ public void TestVTableWithOneInt16()
                 0x9A, 0x78, //value 0
             },
                 builder.DataBuffer.ToFullArray());
+            Google.FlatBuffers.Verifier verifier = new Google.FlatBuffers.Verifier(builder.DataBuffer);
```

Comment:
been a while since I did C#, can we add a using statement to reduce the verbosity?